the_ark/screen_capture.py

```diff
@@ -18,8 +18,9 @@ class Screenshot:
     A helper class for taking screenshots using a Selenium Helper instance
     """
     def __init__(self, selenium_helper, paginated=False, header_ids=None, footer_ids=None,
-                 scroll_padding=DEFAULT_SCROLL_PADDING, pixel_match_offset=DEFAULT_PIXEL_MATCH_OFFSET,
-                 file_extenson=SCREENSHOT_FILE_EXTENSION, resize_delay=0):
+                 content_container_selector="html", scroll_padding=DEFAULT_SCROLL_PADDING,
```

Comment:
Should we add the new param to the end to prevent potential issues caused by positional method calls?